src/database/migrations/20231220235718_removeParticipantIdColumnFromAudit.ts

```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex';
```

Comment:
Sorry - didn't notice this on your last migration, but the file nameconvention appears to be PascalCase for migrations.

Reply:
Is it worth renaming both of them?

Reply:
I'm a bit hesitant to rename the old one because it may recognise it as a new migration and try to run again? Perhaps not, but probably not worth the risk?

Reply:
Yeah that makes sense to me. Was kind of thinking the same thing.